### ISSUE TITLE
Any chance having workspaces on Dropbox?

### ISSUE THREAD

**Issue by JINKEHE** [benweet/stackedit#1336]
Maybe it would be better if there is an option to choose the default workspace?

**Comment by drewdecarme** [comment]
+1 I use Dropbox almost exclusively and try to shy away from Google Sync/Box whatever they call it these days.

**Comment by Bestulo** [comment]
You mentioned on #1342 that this issue (#1336) would solve #1342. You closed #1342, then closed #1336 too. Thus, the issue remained unresolved and Dropbox is not a workspace option. Will this feature be considered?

**Comment by Nthalk** [comment]
@Bestulo that is false. You can save/open individual documents, but not a workspace.

**Comment by charlesangus** [comment]
Bumping this issue, this would be awesome.

**Comment by DaMatrix** [comment]
bumping again, i want to use github and not google drive but it keeps adding itself back and can't be removed...
